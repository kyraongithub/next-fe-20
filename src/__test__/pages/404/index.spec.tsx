import _404 from '@/pages/404';
import { render } from '@testing-library/react';

describe('404 Page', () => {
  it('should render 404 page', () => {
    const { container } = render(<_404 />);
    expect(container).toMatchSnapshot();
  });
});
