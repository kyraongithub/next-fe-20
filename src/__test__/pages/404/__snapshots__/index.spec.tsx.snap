// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 Page should render 404 page 1`] = `
<div>
  <div>
    oops page not found
  </div>
</div>
`;
